src/MSBuild/Resources/xlf/Strings.cs.xlf

```diff
@@ -844,9 +844,6 @@ Když se nastaví na MessageUponIsolationViolation (nebo jeho krátký
                             the console buffer
                         DisableConsoleColor--Use the default console colors
                             for all logging messages.
-                        DisableMPLogging-- Disable the multiprocessor
```

Comment:
there are some remaining resources containing DisableMPLogging , probably should be cleaned up?

Reply:
not sure how VS search works but when I do 
`gh pr checkout https://github.com/dotnet/msbuild/pull/12336`
and search "DisableMPLogging" e.g. in vscode it shows more

Reply:
resolution: these occurences should be updated by Localization pipeline